en/getting-started.md

```diff
@@ -0,0 +1,33 @@
+# Getting Started
+
+Sakurabook is a plugin for pay-as-you-go reservation service which can be used within the Shopify ecosystem. So, if you are a Shopify merchant and you want to boost your online store with the Sakurabook reservation app, you are in the right place.
+
+## Contents
+
+- [Components](#components)
+- [Use Cases](#use-cases)
+
+## Components
+
+The app has two major parts:
+
+- **Sakurabook Admin** - This is an embedded Shopify app in the admin console of your shop, e.g. https://your-store-name.myshopify.com/admin. This part is a merchant-facing user interface.
+
+  ![Alt text](../img/Screenshot%202022-08-28%20at%2010.51.06.png?raw=true "Sakurabook Admin")
+
+- **Sakurabook Booking Wizard** - On the other hand, this is the customer-facing app block that can be added directly to your store via the Shopify Theme customization. It's then accessible on your store, e.g. https://your-store-name.myshopify.com. Once you add it, the booking wizard lets your customers create reservations.
+
+  ![Alt text](../img/Screenshot%202022-08-28%20at%2010.54.43.png?raw=true "Sakurabook Booking Wizard")
+
+Once you've got a good understanding of Shopify and how the apps work, we can have a look at the two main use cases that Sakurabook offers:
```

Comment:
Maybe something like: `Once you've added the plugin in your store` or something like that.